openapi.yaml

```diff
@@ -1,190 +1,432 @@
+      tags:
+        - owners
+      description: Post a new park owner
+      operationId: postOwner
+      parameters:
+      - $ref: '#/parameters/ownerBody'
+      responses:
+        '201':
+          description: Owner account successfully created
+          schema:
+            $ref: '#/definitions/OwnerResult'
+        '400':
+          description: Incorrect syntax in request, or required fields missing
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '409':
+          description: Owner account already exists
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+  /owners/{ownerId}:
+    get:
+      summary: gets the account of a park owner by ID
+      tags:
+        - owners
+      description: gets park owner by ID
+      operationId: getOwnerById
+      parameters:
+      - $ref: '#/parameters/ownerIdPath'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/OwnerResult'
+        '400':
+          description: Incorrect syntax in request, or required fields missing
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '404':
+          description: Owner resource not found
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+    patch:
+      summary: Updates the account of a park owner by ID
+      tags:
+        - owners
+      description: patch park owner by ID
+      operationId: updateOwnerById
+      parameters:
+      - $ref: '#/parameters/ownerIdPath'
+      - $ref: '#/parameters/ownerBody'
+      responses:
+        '200':
+          description: Owner successfully updated
+          schema:
+            $ref: '#/definitions/OwnersResult'
+        '400':
+          description: Incorrect syntax in request, or required fields missing
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '404':
+          description: Owner resource not found
+          schema:
+            $ref: '#/definitions/ErrorResult'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/ErrorResult'
+    delete:
+      summary: Delete a park owner account by ID
+      tags:
+        - owners
+      description: Delete owner by ID
+      operationId: deleteOwnerById
+      parameters:
+      - $ref: '#/parameters/ownerIdPath'
+      responses:
+        '204':
+          description: Park successfully deleted
+        '404':
+          description: Owner resource not found
           schema:
             $ref: '#/definitions/ErrorResult'
         '500':
           description: Internal server error
           schema:
             $ref: '#/definitions/ErrorResult'
```

Comment:
This 500 response is used multiple times. Apparently, you can define response schemas in a top-level `responses` field: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#fixed-fields
You may want to take advantage of this to avoid repetition. This could probably be done with other responses as you see fit.